Adyen/Service/Payout/ReviewingService.cs

```diff
@@ -0,0 +1,86 @@
+/*
+* Adyen Payout API
+*
+*
+* The version of the OpenAPI document: 68
+* Contact: developer-experience@adyen.com
+*
+* NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+* https://openapi-generator.tech
+* Do not edit the class manually.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Adyen.Constants;
+using Adyen.Model;
+using Adyen.Service.Resource;
+using Adyen.Model.Payout;
+using Newtonsoft.Json;
+
+namespace Adyen.Service.Payout
+{
+    /// <summary>
+    /// Represents a collection of functions to interact with the API endpoints
+    /// </summary>
+    public class ReviewingService : AbstractService
+    {
+        private readonly string _baseUrl;
+        
+        public ReviewingService(Client client) : base(client)
+        {
+            _baseUrl = client.Config.Endpoint + "/pal/servlet/Payout/" + ClientConfig.PayoutApiVersion;
```

Comment:
Double-checking here, how do we distinguish between the live and test endpoints?

Reply:
client.config.Endpoint is an config object set when constructing the Client object and either set to the live or test endpoint of the specific endpoint url.